serverjs/updatecards.js

```diff
@@ -687,7 +689,15 @@ function convertCard(card, isExtra) {
   newcard.color_identity = newcard.color_identity.concat(card.color_identity);
   newcard.set = card.set;
   newcard.collector_number = card.collector_number;
-  newcard.promo = card.promo;
+  newcard.promo =
```

Comment:
I would put the promo detection in its own function to match the style of the `convertThing()` attributes and keep `convertCard()` as small as possible.

Reply:
I think I value not having these fields in the card object, to reduce the size of the actual object a lot more.